test/sequential/test-vm-timeout-rethrow.js

```diff
@@ -6,14 +6,10 @@ const spawn = require('child_process').spawn;
 
 if (process.argv[2] === 'child') {
   const code = 'let j = 0;\n' +
```

Comment:
I left it intentionally so it can be used in return, but with `while(true);` won't be necessary. Will remove it.